src/sparselm/model/__init__.py

```diff
@@ -22,8 +21,10 @@
     RidgedBestSubsetSelection,
 )
 from ._ols import OrdinaryLeastSquares
+from ._stepwise import StepwiseEstimator
```

Comment:
Can you move `StepwiseEstimator` to another module? Since it is not an estimator by itself, it should be imported differently than the standard estimators. It can either be directly from `sparselm.stepwise` or following sklearn we can define a `compose` module and have it there so that it is imported as `from sparselm.compose import StepwiseEstimator`.

This will also make a separation of concern for unit-tests.

Reply:
Done.